ipaserver/plugins/pkinit.py

```diff
@@ -57,49 +42,22 @@ class pkinit(Object):
     label=_('PKINIT')
 
 
-def valid_arg(ugettext, action):
-    """
-    Accepts only Enable/Disable.
-    """
-    a = action.lower()
-    if a != 'enable' and a != 'disable':
-        raise errors.ValidationError(
-            name='action',
-            error=_('Unknown command %s') % action
-        )
-
 @register()
 class pkinit_anonymous(Command):
-    __doc__ = _('Enable or Disable Anonymous PKINIT.')
-
-    princ_name = '%s@%s' % (ANON_USER, api.env.realm)
-    default_dn = DN(('krbprincipalname', princ_name), ('cn', api.env.realm), ('cn', 'kerberos'), api.env.basedn)
+    __doc__ = _('Originally to enable or disable Anonymous PKINIT.\n'
+                'Deprecated since FreeIPA 4.5.1')
 
     takes_args = (
-        Str('action', valid_arg),
+        Str('action?'),
```

Comment:
True if it is deprecated and no-op, then we should not bother users about required options. Do you agree @HonzaCholasta ?